packager/react-packager/src/DependencyResolver/fastfs.js

```diff
@@ -297,7 +297,7 @@ class File {
     Object.keys(this.children).forEach(key => {
       const file = this.children[key];
       if (file.isDir) {
-        files.push(...file.getFiles());
+        file.getFiles().forEach((f) => files.push(f));
```

Comment:
Maybe `files.push.apply(files, file.getFiles())` to avoid the overhead of invoking the arrow function each time? (also would be good to have a comment explaining that this micro optimization is meaningful)